Lib/fontMath/mathKerning.py

```diff
@@ -219,7 +219,7 @@ def _processMathTwo(self, factor, funct):
         for k, v in self._kerning.items():
             v = funct(v, factor)
             kerning[k] = v
-        ks = MathKerning(kerning)
+        ks = MathKerning(kerning, deepcopy(self._groups))
```

Comment:
if you pass the `groups` parameter to the MathKerning constructor, then the `updateGroups` method, called in the constructor, will populate the `_side1Groups` and `_side2Groups` accordingly, so you don't need to set them again in the following lines.

Also, I noticed that the MathKerning `__init__` method has keyword arguments which default to mutable dicts:
http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments

would be better if they default to None, and then in the `__init__` we set the attributes to new empty dict instances if parameter is None.